tests/integration/test_search_support_docs_tool.py

```
@@ -0,0 +1,553 @@
+        """Test search with multiple filters applied."""
+        with (
+            patch("src.tools.support_documentation_tools.EmbeddingService") as mock_embedding_service,
+            patch("src.tools.support_documentation_tools.SupportDocumentationService") as mock_support_service,
+        ):
+            # Setup mocks
+            mock_embedding_instance = AsyncMock()
+            mock_embedding_service.return_value = mock_embedding_instance
+
+            mock_service_instance = AsyncMock()
+            mock_service_instance.search_support_documents.return_value = sample_search_results
+            mock_service_instance._prepare_search_filters.return_value = {
```

Pylint (3.3.7)

[warning] Access to a protected member _prepare_search_filters of a client class

(W0212)